### PR TITLE
update android release-notes for v2.5.0

### PR DESCRIPTION
**Pull request by vonageCI**

### CODE REVIEW

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,144 @@ navigation_weight: 0
...
+   }
+});
+
+```
+
+## Version 2.3.0 - February 11, 2020
+
+### Added
+
+- Add `updateAsDelivered` and `updateAsSeen` to `NexmoAttachmentEvent` & `NexmoTextEvent` as helper method
```
Comment (tbedford) [inline _documentation/en/client-sdk/sdk-documentation/android/release-notes.md]: Do not use ampersand here, use 'and'.

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md [hunk 2]
```diff
@@ -6,6 +6,144 @@ navigation_weight: 0
...
+
+       }
+   })
+```
+
+### Fixed
+- fix `markAsSeen` and `markAsDelivered` for `NexmoTextEvent` and `NexmoAttachmentEvent`
+
+### Changed
+- upgrade dependency libraries please add to your build gradle
```
Comment (tbedford) [inline _documentation/en/client-sdk/sdk-documentation/android/release-notes.md]: gradle -> Gradle. Would it be clearer as "Gradle build file"?

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md [hunk 3]
```diff
@@ -6,6 +6,144 @@ navigation_weight: 0
...
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
+``` 
+
+## Version 2.2.0 - January 31, 2020
+
+### Added
+
+- Add support for Custom Push Notifications, using `processNexmoPush()` (`processPushNotification()` is deprecated)
```
Comment (tbedford) [inline _documentation/en/client-sdk/sdk-documentation/android/release-notes.md]: Use of brackets here: (`processPushNotification()` is deprecated) is confusing. Remove brackets and just start a new sentence.

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md [hunk 4]
```diff
@@ -6,6 +6,144 @@ navigation_weight: 0
...
+       override fun onSuccess(result: Any?) {
+       Log.d(TAG, TAG + "onTextEvent.markAsDelivered():onSuccess with: " + result.toString())
+       }
+       override fun onError(error: NexmoApiError) {
+       Log.d(TAG, TAG + "onTextEvent.markAsDelivered():onError with: " + error)
+        }
+   })
+   ```
+   
+ - add `markAsSeen` method to `NexmoTextEvent` and `NexmoAttachmentEvent`
```
Comment (tbedford) [inline _documentation/en/client-sdk/sdk-documentation/android/release-notes.md]: add -> Add

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md (hunk 1 above)
Comment (apassal) [reply _documentation/en/client-sdk/sdk-documentation/android/release-notes.md]: done

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md (hunk 2 above)
Comment (apassal) [reply _documentation/en/client-sdk/sdk-documentation/android/release-notes.md]: done

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md (hunk 3 above)
Comment (apassal) [reply _documentation/en/client-sdk/sdk-documentation/android/release-notes.md]: done

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md (hunk 4 above)
Comment (apassal) [reply _documentation/en/client-sdk/sdk-documentation/android/release-notes.md]: done

**Pull request merged**